backend/src/abilities/tests/ability-fixture.ts

```diff
@@ -71,6 +88,48 @@ export function createAbilityFixture(): AbilityFixture {
         .withUpdatedAt(currentDate);
 
       return builder.build();
+    },
+    
+    // Result and error handling
+    getResult(): any {
+      return result;
+    },
+    
+    setResult(newResult: any): void {
```

Comment:
Code Review: The 'any' type for 'newResult' is too permissive. Consider using a more specific type or a generic type parameter to improve type safety.